<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefDialog</class>
 <widget class="QDialog" name="PrefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="prefGeneralTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="autoRefreshCheckBox">
           <property name="text">
            <string>Automatically refresh tables every</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="autoRefreshIntervalDoubleSpinBox">
           <property name="suffix">
            <string> seconds</string>
           </property>
           <property name="maximum">
            <double>179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="randomizeWindowTitlesCheckBox">
         <property name="text">
          <string>Randomize Evanesco window titles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="randomizeTrayIconCheckBox">
         <property name="text">
          <string>Randomize Evanesco tray icon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hideFromScreenCaptureCheckBox">
         <property name="text">
          <string>Hide Evanesco from screen capture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hideTaskbarIconCheckBox">
         <property name="text">
          <string>Hide taskbar icon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="minimizeToTrayCheckBox">
         <property name="text">
          <string>Minimize to tray icon</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>168</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="prefPayloadTab">
      <attribute name="title">
       <string>Payload</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="randomizeDllFileNameCheckBox">
         <property name="text">
          <string>Randomize DLL file name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hideTargetTaskbarIconsCheckBox">
         <property name="text">
          <string>Hide target window taskbar icons</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="perfAutohideTab">
      <attribute name="title">
       <string>Autohide</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="enableAutohideCheckBox">
         <property name="text">
          <string>Enable autohide</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hideAutohideProcessesOnStartCheckBox">
         <property name="text">
          <string>Hide processes on the autohide list when Evanesco starts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autohideNotifyCheckBox">
         <property name="text">
          <string>Send a notification when a window got automatically hidden (requires tray icon)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Maximum time to wait for process window creation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maxWindowCreationWaitMsSpinBox">
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="maximum">
            <number>2147483647</number>
           </property>
           <property name="value">
            <number>3000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Autohide List</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="autohideFileNameLineEdit">
              <property name="placeholderText">
               <string>Enter file name or full path (e.g., notepad.exe)</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="autohideSelectFilePushButton">
              <property name="text">
               <string>Select File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="autohideAddPushButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::ListAdd"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="autohideRemovePushButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::ListRemove"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="autohideListWidget"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="resetPushButton">
         <property name="text">
          <string>Reset all preferences</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrefDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrefDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
